### ISSUE TITLE
grpc-trace-bin header support

### ISSUE THREAD

**Issue by xd009642** [open-telemetry/opentelemetry-rust-contrib#37]
As well as `traceparent` for context propagation there's also `grpc-trace-bin` and `x-cloud-trace-context`, my current work system is using `grpc-trace-bin` so it would be nice if it was as easy to propagate the traces as it was with `traceparent` or the b3 headers

**Comment by RichardChukwu** [comment]
Hi @xd009642 @hdost Is this being actively worked on at the moment please?

**Comment by xd009642** [comment]
No, but I'm also not sure if it's relevant anymore? It might have gotten in elsewhere and this issue just not been cleaned up

**Comment by RichardChukwu** [comment]
Ok. How do we confirm that?
